core/controllers/suggestion.py

```diff
@@ -297,7 +297,12 @@ class ReviewableSuggestionsHandler(SuggestionsProviderHandler):
 
     @acl_decorators.can_view_reviewable_suggestions
     def get(self, target_type, suggestion_type):
-        """Handles GET requests."""
+        """Handles GET requests.
```

Comment:
@hardikkat24, Can you please cover one (or multiple) files to have all the docstring in a PR? (I see a lot of functions in this file needs docstring!)

Reply:
@DubeySandeep I have opened a new issue for this(#11776 ). I think that this PR can be closed and completion of documentation can be handled in that issue in a well-ordered manner.

Reply:
Done